mlfinlab/tests/test_labels.py

```diff
@@ -292,3 +298,33 @@ def test_drop_labels(self):
         # Assert that threshold works
         new_labels = drop_labels(events=triple_labels, min_pct=0.20)
         self.assertTrue(0 in set(new_labels['bin']))
+
+    def test_trend_scanning_labels(self):
```

Comment:
About how long does this take to run? 

There are actually a couple of unit tests in here. Should we not rather have them in their own test method with a docstring explaining what they test for?

Reply:
I am nervous that it adds another few minutes to our already long build times.

Reply:
Let me check how long the test takes

Reply:
The test takes 1s ;)